src/openforms/registrations/contrib/email/config.py

```diff
@@ -21,7 +21,19 @@ class EmailOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
     payment_emails = serializers.ListField(
         child=serializers.EmailField(),
         label=_(
-            "The email addresses to which the payment status update will be sent (defaults to general registration addresses)"
+            "The email addresses to which the payment status update will be sent "
+            "(defaults to general registration addresses)"
         ),
         required=False,
     )
+    attach_files_to_email = serializers.BooleanField(
+        label=_("attach files to email"),
+        allow_null=True,
+        default=None,  # falls back to the global default
+        help_text=_(
+            "Enable to attach file uploads to the registration email. Note that this "
+            "is the global default which may be overridden per form. Form designers "
+            "should take special care to ensure that the total file upload sizes do "
+            "not exceed the email size limit."
```

Comment:
This is currently also the help_text that is shown within the form registration tab
![Screenshot from 2022-02-22 15-30-47](https://user-images.githubusercontent.com/19154114/155153254-acd403bd-a884-4a70-b356-e432d8c5074d.png)